### PR TITLE
WPF-62607 update how to in sf skinmanager

### PR DESCRIPTION
**Pull request by Kayalvizhi-Sivashanmugam**
Added the following topics in the "How to" section:

How to 

-    Apply custom theme
-    Switch theme at runtime
-    Switch theme with custom styles

### CODE REVIEW

File: wpf/Themes/Skin-Manager.md [hunk 1]
```diff
@@ -939,83 +939,93 @@ private void Window_Closed(object sender, EventArgs e)
...
+### Customize theme in application level
 
-The naming convention of a control style will be like `Syncfusion-ControlName-Style`. For example, `MaterialDarkButtonAdvStyle`.
+To customize the Syncfusion theme in application level, merge the theme in ResourceDictionary and override the style using BasedOn property. 
 
-The following steps explain how to override the Syncfusion themes:
+In ResourceDictionary, mention the style path which need to be override, in BasedOn, mention the key stated in the table.
 
-**Step1:**
+We can customize or override the theme styles by following the steps outlined below.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:971]: Explain that we going to show example for button

File: wpf/Themes/Skin-Manager.md [hunk 2]
```diff
@@ -939,83 +939,93 @@ private void Window_Closed(object sender, EventArgs e)
...
 
-Add respective resource dictionary from the themes assembly to the application.
 
-The resource dictionary will be in this format: `/Syncfusion.Themes.<ThemeName>.WPF;component/<ControlName>/<ControlName>.xaml`, where `ThemeName` denotes the theme name for which overridden style is going to be applied and `ControlName` denotes the control name. For example, `/Syncfusion.Themes.MaterialDark.WPF;component/ButtonAdv/ButtonAdv.xaml`.
+**Step 1**: Adding ResourceDictionary for Framework button.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:974]: Explain in generic and then show button example in all steps

File: wpf/Themes/Skin-Manager.md [hunk 3]
```diff
@@ -1072,6 +1082,105 @@ Themes for application can be changed at runtime by changing `VisualStyle` prope
...
+
+### Switch theme with custom styles
+
+To change a theme dynamically with custom styles, utilize the resource dictionary and override the style using the 'BasedOn' property.
+
+We have provided an example that demonstrates switching between the Window11Light and Windows11Dark themes using the SfGrid control. In this example, we customize the foreground color of the GridTableSummaryCell to green in Window11Light and red in Window11Dark.
+Always call the SetTheme() method of the SfSkinManager when changing themes.
+
+ 
+*  **Step 1**: In this example, we have an SfDataGrid and a combobox, so we need to add resource dictionaries with paths in merged dictionaries in ViewModel or code behind.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1104]: explain ComboBox is for changing theme

File: wpf/Themes/Skin-Manager.md [hunk 4]
```diff
@@ -1072,6 +1082,105 @@ Themes for application can be changed at runtime by changing `VisualStyle` prope
...
+
+{% tabs %}
+
+{% highlight C# %}
+
+       Application.Current.Resources.MergedDictionaries.Add(new ResourceDictionary()
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1110]: Add ComoboBox selection changed

File: wpf/Themes/Skin-Manager.md [hunk 5]
```diff
@@ -1072,6 +1082,105 @@ Themes for application can be changed at runtime by changing `VisualStyle` prope
...
+
+*  **Step 2**: Customize the foreground color and font weight of the table summary row in Both Light and dark themes.
+
+
+
+
+### Light Theme:
+{% tabs %}
+
+{% highlight C# %}
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1140]: XAML.
Add

File: wpf/Themes/Skin-Manager.md [hunk 6]
```diff
@@ -939,83 +939,97 @@ private void Window_Closed(object sender, EventArgs e)
...
-The following steps explain how to override the Syncfusion themes:
+In ResourceDictionary, mention the style path which need to be override, in BasedOn, mention the key stated in the table.
 
-**Step1:**
+We can customize or override the theme styles by following the steps outlined below.
 
-Add respective resource dictionary from the themes assembly to the application.
 
-The resource dictionary will be in this format: `/Syncfusion.Themes.<ThemeName>.WPF;component/<ControlName>/<ControlName>.xaml`, where `ThemeName` denotes the theme name for which overridden style is going to be applied and `ControlName` denotes the control name. For example, `/Syncfusion.Themes.MaterialDark.WPF;component/ButtonAdv/ButtonAdv.xaml`.
+**Step 1**: Include the resource dictionaries of the controls(Which are used in our application) into the merged dictionaries. In this example, we added ‘ResourceDictionary’ for Button.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:974]: Step 1: Merge the resource dictionaries of the controls which need to customize. In this example, we have merged the Framework Button‘s resource dictionary.

File: wpf/Themes/Skin-Manager.md [hunk 7]
```diff
@@ -939,83 +939,97 @@ private void Window_Closed(object sender, EventArgs e)
...
+     <ResourceDictionary Source="/Syncfusion.Themes.Windows11Light.WPF;component/MSControl/Button.xaml" /> 
 </ResourceDictionary.MergedDictionaries>
-</ResourceDictionary>
 
 {% endhighlight %}
 
 {% endtabs %}
 
-**Step2:**
+**Step 2**: : Customize the style of controls. In this step, we can customize the style of the Button. The code below customizes the background color, text color, font, and border color of the Button.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:988]: Step 2: Declare style for the control with BasedOn key. Here style of the button has been declared and its key is used in BasedOn. Button's Background, Foreground, FontFamily, BorderBrush has been overridden.

File: wpf/Themes/Skin-Manager.md [hunk 8]
```diff
@@ -939,83 +939,97 @@ private void Window_Closed(object sender, EventArgs e)
...
+         <Setter Property="BorderBrush" Value="#5dbea3" />   
+   </Style>
+
+{% endhighlight %}
 
-Define the new style using the `BasedOn` property.
+{% endtabs %}
 
-The following code sample overrides the Syncfusion style for the `ButtonAdv` Control.
+**Step 3**: : Now, utilize the customized styles in our application using the 'Style' property. This step adds custom styles of Buttons in WPF application. The code below creates a ‘StackPanel’ with two buttons:
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1009]: remove: The code below creates a ‘StackPanel’ with two buttons:

File: wpf/Themes/Skin-Manager.md [hunk 9]
```diff
@@ -1072,6 +1086,142 @@ Themes for application can be changed at runtime by changing `VisualStyle` prope
...
 {% endtabs %}
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/change-themes-at-runtime-using-skinmanager).
+
+
+### Switch theme with custom styles
+
+To change a theme dynamically with custom styles, utilize the resource dictionary and override the style using the 'BasedOn' property.
+
+We have provided an example that demonstrates switching between the Window11Light and Windows11Dark themes using the SfGrid control. In this example, we customize the foreground color of the GridTableSummaryCell to green in Window11Light and red in Window11Dark.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1104]: **Window 11 Light** and **Windows 11 Dark**

File: wpf/Themes/Skin-Manager.md [hunk 10]
```diff
@@ -1072,6 +1086,142 @@ Themes for application can be changed at runtime by changing `VisualStyle` prope
...
+
+
+ 
+*  **Step 1**: In this example, we have an SfDataGrid and a RadioButton which is used to switching theme and we need to add resource dictionaries with paths in merged dictionaries in ViewModel or code behind. The following code can be used to effortlessly switch themes.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1108]: ViewModel -> view model

File: wpf/Themes/Skin-Manager.md [hunk 11]
```diff
@@ -1072,6 +1086,142 @@ Themes for application can be changed at runtime by changing `VisualStyle` prope
...
+{% tabs %}
+
+{% highlight C# %}
+
+      
+
+      private void RadioButton_Click(object sender, RoutedEventArgs e)
+        {
+            var themeName = (sender as RadioButton).Content.ToString();
+            string syncfusionTheme = null;
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1120]: string.Empty()

File: wpf/Themes/Skin-Manager.md [hunk 12]
```diff
@@ -1072,6 +1086,142 @@ Themes for application can be changed at runtime by changing `VisualStyle` prope
...
+            {
+                Source = new Uri($"/WpfApp1;component/Themes/{selectedTheme}.xaml", UriKind.RelativeOrAbsolute)
+            });
+        }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+*  **Step 2**: Customize the foreground color and font weight of the table summary row in Both Light and dark themes.
```
Comment (jeganraj-m) [inline wpf/Themes/Skin-Manager.md:1172]: check caps

**Review by jeganraj-m — APPROVED** [review]
changes are fine

File: wpf/Themes/Skin-Manager.md (hunk 1 above)
Comment (Kayalvizhi-Sivashanmugam) [reply wpf/Themes/Skin-Manager.md:971]: Mentioned query was addressed.

File: wpf/Themes/Skin-Manager.md (hunk 7 above)
Comment (Kayalvizhi-Sivashanmugam) [reply wpf/Themes/Skin-Manager.md]: Mentioned query was addressed.

File: wpf/Themes/Skin-Manager.md (hunk 6 above)
Comment (Kayalvizhi-Sivashanmugam) [reply wpf/Themes/Skin-Manager.md]: Mentioned query was addressed.

**Pull request merged**